Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqProject.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GcbqProject" {
+
+	It "should return the list of projects that the user has permissions to view" {
+		$batches = Get-GcbqProject
```

Comment:
I don't think we can. I believe every project needs to be billable to someone. Dynamically creating billable entities seems like a no-go.